### PR TITLE
Issue#23/charge nets payment

### PR DESCRIPTION
**Pull request by markiyanshvets**

### CODE REVIEW

File: Api/BccPay.Core.Infrastructure/PaymentProviders/Implementations/Nets/NetsPaymentProvider.cs [hunk 1]
```diff
@@ -153,6 +154,25 @@ public Task<IPaymentResponse> GetPayment(string paymentId)
...
+                     netsStatusDetails.PaymentCheckoutId,
+                     new NetsChargeRequest
+                     {
+                         Amount = Convert.ToInt32(payment.Amount * 100)
+                     });
+
+                netsStatusDetails.InvoiceId = invoice?.Invoice?.InvoiceNumber;
+            }
+            catch (ApiException exception)
+            { }
```
Comment (yuriy-butkevych) [inline Api/BccPay.Core.Infrastructure/PaymentProviders/Implementations/Nets/NetsPaymentProvider.cs:173]: Should we add the exception to attempt errors property?

File: Api/BccPay.Core.Infrastructure/PaymentProviders/Implementations/Nets/NetsPaymentProvider.cs [hunk 2]
```diff
@@ -153,6 +154,25 @@ public Task<IPaymentResponse> GetPayment(string paymentId)
...
+        public async Task ChargePayment(Payment payment, Attempt attempt)
+        {
+            var netsStatusDetails = (NetsStatusDetails)attempt.StatusDetails;
+            try
+            {
+                var invoice = await _netsClient.ChargePayment(_headers,
+                     netsStatusDetails.PaymentCheckoutId,
+                     new NetsChargeRequest
+                     {
+                         Amount = Convert.ToInt32(payment.Amount * 100)
```
Comment (yuriy-butkevych) [inline Api/BccPay.Core.Infrastructure/PaymentProviders/Implementations/Nets/NetsPaymentProvider.cs:167]: Should we charge x 100 amount here?

File: Api/BccPay.Core.Infrastructure/PaymentProviders/Implementations/Nets/NetsPaymentProvider.cs (hunk 1 above)
Comment (markiyanshvets) [reply Api/BccPay.Core.Infrastructure/PaymentProviders/Implementations/Nets/NetsPaymentProvider.cs:173]: yes, I think, refit throw exception on 4xx response code
in any case, we need to update the attempt status with the webhook

File: Api/BccPay.Core.Infrastructure/PaymentProviders/Implementations/Nets/NetsPaymentProvider.cs (hunk 2 above)
Comment (markiyanshvets) [reply Api/BccPay.Core.Infrastructure/PaymentProviders/Implementations/Nets/NetsPaymentProvider.cs:167]: yes, nets provider amount is an integer (monets), and we store this data in decimal

**Pull request merged**